enrolments/tests/views/test_class.py

```diff
@@ -33,11 +33,11 @@ def setUp(self):
             facilitator_id=self.user.id,
             attendance=[{"date": "2020-01-01", "attendees": [self.student1.id]}],
         )
-        self.empty_class = Class.objects.create(
-            name="Test Empty Class",
-            session_id=self.session2.id,
-            facilitator_id=self.user.id,
-            attendance=[{"date": "2020-01-01", "attendees": []}],
+        self.class_create = Class.objects.create(
+            name="Test Class Create",
+            days=[Class.MONDAY, Class.WEDNESDAY],
+            location="Waterloo",
+            facilitator=self.user,
         )
```

Comment:
we probably don't need to actually create a class object here, maybe we can pass these values directly in the payload on line 77 below?